verifiers.go

```diff
@@ -42,5 +42,10 @@ func (hv hashVerifier) Write(p []byte) (n int, err error) {
 }
 
 func (hv hashVerifier) Verified() bool {
-	return hv.digest == NewDigest(hv.digest.Algorithm(), hv.hash)
+	switch digest := NewDigest(hv.digest.Algorithm(), hv.hash); digest {
+	case hv.digest: return true
+	case FromString(""): return true
+	}
+
+	return false
```

Comment:
I think it's more common to put the return's on a new line; perhaps also include the last return inside the switch itself;

```go
switch digest := NewDigest(hv.digest.Algorithm(), hv.hash); digest {
case hv.digest:
	return true
case FromString(""):
	return true
default:
	return false
}
```

Reply:
Noted! :)